modules/ROOT/partials/configuration/tableofcontents_orderedlist.adoc

```diff
@@ -7,7 +7,7 @@ The `+tableofcontents_orderedlist+` option allows Tables of Contents to be rende
 
 When the `+tableofcontents_orderedlist+` option is set to `+true+`, Tables of Contents are rendered as numeric ordered lists.
 
-To customise the type of ordered list, add the `+xref:tableofcontents_orderedlist_type[tableofcontents_orderedlist_type]+` option to the configuration.
+To customize the type of ordered list, add the xref:tableofcontents_orderedlist_type[`+tableofcontents_orderedlist_type+`] option to the configuration.
```

Comment:
Suggestion to remove `Literal Monospace` formatting so the link to the option is visible without hover.
```suggestion
To customize the type of ordered list, add the xref:tableofcontents_orderedlist_type[tableofcontents_orderedlist_type] option to the configuration.
```